ksql-common/src/main/java/io/confluent/ksql/util/SchemaUtil.java

```diff
@@ -238,8 +232,8 @@ public static String getJavaCastString(Schema schema) {
 
   public static Schema addImplicitRowTimeRowKeyToSchema(Schema schema) {
     SchemaBuilder schemaBuilder = SchemaBuilder.struct();
-    schemaBuilder.field(SchemaUtil.ROWTIME_NAME, Schema.INT64_SCHEMA);
-    schemaBuilder.field(SchemaUtil.ROWKEY_NAME, Schema.STRING_SCHEMA);
+    schemaBuilder.field(SchemaUtil.ROWTIME_NAME, Schema.OPTIONAL_INT64_SCHEMA);
```

Comment:
This is a bigger change that I will make in an separate PR which is switching to OPTIONAL schemas. If the schema is not optional we cannot have `null` values for the column/field.
Before we didn't need this since we were handling the data ourselves. Now that we use connect data format in order to be able to have null for ROWKEY/ROWTIME we should have OPTIONAL schema for them.

Reply:
OK